README.md

```diff
@@ -0,0 +1,150 @@
+# GLaDOS
+
+## Generic Language and Data Operand Syntax
+
+### 🔗 Useful Links
+
+- 🎨 Design & Architecture: [Figma Board](https://www.figma.com/design/nEfFaG3XVqWuiBLY3HY8iH/)
+- 📋 Project Management: [GitHub Project](https://github.com/orgs/Sigmapitech/projects/15)
+- 📚 BNF Grammar: [Railroad Diagram Generator](https://www.bottlecaps.de/rr/ui)
+- 🧪 BNF Playground: [Online Tester](https://bnfplayground.pauliankline.com/)
+
+### 📋 Overview
+
+GLaDOS is a functional programming language designed and implemented in Haskell. It features a custom LISP-like syntax (bootstrap), a stack-based virtual machine, and a compiler.
```

Comment:
Bootstrap is a specific term in programming languages, which might cause confusion (bootstrap is the step to compile a language in itself leaving another pre existing dependency)